WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -746,7 +769,7 @@ def __startJobAgent(self):
 
     retCode, _output = self.executeAndGetOutput( jobAgent, self.pp.installEnv )
     if retCode:
-      self.log.error( "Could not start the JobAgent" )
+      self.log.error( "Error executing the JobAgent [ERROR %d]" % retCode )
```

Comment:
Maybe we could try and print a bit more information here like a "ps" of the process group, just to see what is still running and what now?